src/app/servicesModule/services/create-poll-storage.service.ts

```diff
@@ -51,10 +51,18 @@ export class CreatePollStorageService {
     const connectionConfig = ConnectionConfig.createWithLocalIpfsConnection(
       new BlockchainNetworkConnection(blockChainNetworkType, blockChainHost, blockChainPort, blockChainProtocol),
       new IpfsConnection(storageHost, storagePort, storageOptions));
-
-    this.uploader = new Uploader(connectionConfig);
+    const storageHostUploader = environment.storageConnectionUnload.host;
+    const storagePortUploader = environment.storageConnectionUnload.port;
+    const storageOptionsUploader = environment.storageConnectionUnload.options;
+    const connectionConfigUploader = ConnectionConfig.createWithLocalIpfsConnection(
+      new BlockchainNetworkConnection(blockChainNetworkType, blockChainHost, blockChainPort, blockChainProtocol),
+      new IpfsConnection(storageHostUploader, storagePortUploader, storageOptionsUploader));
+    this.uploader = new Uploader(connectionConfigUploader);
     this.searcher = new Searcher(connectionConfig);
     this.downloader = new Downloader(connectionConfig);
+
+    console.log('downloader', this.downloader)
```

Comment:
Not all browsers support `console.log` and we should only use it or enable it if we're debugging. Please remove.